jest.config.js

```diff
@@ -1,8 +1,9 @@
 module.exports = {
   collectCoverageFrom: [
-    'app/javascript/src/**/*.{js,jsx}',
+    'app/javascript/**/*.{js,jsx}',
```

Comment:
When this config was updated, it was a couple of years ago where the thought was components and most frontend source would go under the `src` folder, but the reality is it's all under `app/javascript`.